index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</heads>
+<body>
+   <h1>My very first Microverse project: Hello Microverse</h1> 
```

Comment:
Great job @AndisiweJ 🤝

- [ ] [Optional] Just like the previous reviewer suggested, kindly add a class to your element and then apply styles to this element through this class in the stylesheet.